templates/undercloud.conf.j2

```diff
@@ -11,6 +11,7 @@ container_insecure_registries={{ registry_mirror }},{{ insecure_registries }}
 undercloud_timezone = UTC
 undercloud_public_host = {{ undercloud_public_host }}
 undercloud_admin_host = {{ undercloud_admin_host }}
+clean_nodes = true
```

Comment:
Instead, we can set the clean_debug to true by default in [group_vars/all.yml](https://github.com/redhat-performance/jetpack/blob/master/group_vars/all.yml#L121) and disable the node cleaning if needed by setting it to false.
This way there is no need for setting clean_nodes to true again. Thoughts?

Reply:
Oh sweet, I didn’t even check that we already have that variable. Yeh let me just change the default there. Good point.